frontend/src/theme/primitives/sizes.ts

```diff
@@ -0,0 +1,16 @@
+export const sizes = {
+  full: "100%",
+  sidebar: "21rem",
+  "3xs": "14rem",
+  "2xs": "16rem",
+  xs: "20rem",
+  sm: "24rem",
+  md: "28rem",
+  lg: "32rem",
+  xl: "36rem",
+  "2xl": "42rem",
+  "3xl": "48rem",
+  "4xl": "56rem",
+  "5xl": "64rem",
+  "6xl": "72rem",
```

Comment:
should we apply this naming style with the 3xs, 6xl to the spacing variables? should just be consistent but fine with either

Reply:
oh yea. Good point. Will do.